src/main/util.ts

```diff
@@ -451,6 +451,16 @@ const cutVideo = async (initialFile: string, finalDir: string, desiredDuration:
     });
 }
 
+/**
+ * Gets string value from the config in a more reliable manner.
+ * @param cfg the config store
+ * @param preference the preference
```

Comment:
preference as a var name is a bit unfamiliar to me in this context, I was expecting `attribute` or something similar, but pleased to be educated if this is a common thing?

Reply:
well, i was thinking preference because it is a user pref they are setting. im fine with attribute too.